index.js

```diff
@@ -22,10 +22,17 @@ const getCSS = () => got('https://github.com').then(response => {
 	return ret;
 });
 
-const getRenderedFixture = () => got(fixtureURL).then(response => {
-	const $ = cheerio.load(response.body);
-	return $('.markdown-body').parent().html();
-});
+const getRenderedFixture = () => got.post('https://api.github.com/markdown', {
+	headers: {
+		'content-type': 'application/json',
+		'user-agent': 'generate-github-markdown-css'
+	},
+	body: JSON.stringify({
+		mode: 'gfm',
+		context: 'sindresorhus/generate-github-markdown-css',
+		text: fs.readFileSync('./fixture.md', 'utf8')
```

Comment:
This needs use `path.join(__dirname, 'fixture.md')`. Using `process.cwd()` would result in the file not being found when it's called from anywhere else.

Reply:
Done.